web/src/dropdown_widget.ts

```diff
@@ -239,13 +239,22 @@ export class DropdownWidget {
     }
 
     setup(): void {
+        let keyboard_action: boolean;
```

Comment:
This is not a good name. It's not clear what it stands for. Also, should this be tracked as a class level variable with a comment?